course/src/main/res/drawable/course_ic_not_supported_block.xml

```diff
@@ -3,21 +3,6 @@
     android:height="100dp"
```

Comment:
1- can we use white color as a strokeColor to make it more generic?
2- requires auto-format